Strategic/Quests.h

```diff
@@ -756,10 +756,13 @@ extern BOOLEAN CheckForNewShipment( void );
 extern BOOLEAN CheckTalkerFemale( void );
 extern BOOLEAN CheckTalkerUnpropositionedFemale( void );
 
+enum PowQuestState
+{
+	Q_FAIL,
+	Q_SUCCESS,
+	Q_RESET,
+	Q_END,
+	Q_LEFT_SECTOR
+};

```

Comment:
consider using `enum class` for this if you don't need them to implicity convert to int

Reply:
I'm getting identifier "Q_END" is undefined errors in strategicmap.cpp after changing this to enum class